<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepoSettingsDialog</class>
 <widget class="QDialog" name="RepoSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Repo Settings for {repo}</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nicknameLabel">
     <property name="text">
      <string>Repo Nic&amp;kname:</string>
     </property>
     <property name="buddy">
      <cstring>nicknameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nicknameEdit">
     <property name="toolTip">
      <string>This nickname will appear within {app} in tab names, menus, etc. It does not change the actual name of the repoâ€™s directory. Leave blank to clear the nickname.</string>
     </property>
     <property name="placeholderText">
      <string>No nickname</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="localIdentityCheckBox">
     <property name="text">
      <string>Create commits under a custom &amp;identity in this repo</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="emailLabel">
     <property name="text">
      <string>&amp;Email:</string>
     </property>
     <property name="buddy">
      <cstring>emailEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="emailEdit"/>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RepoSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RepoSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localIdentityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>nameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localIdentityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>nameEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localIdentityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>emailLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>localIdentityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>emailEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
